contrib/lang/haskell/packages.el

```diff
@@ -196,8 +195,7 @@
     :if (configuration-layer/layer-declaredp 'company-mode)
     :init
     (progn
-      (add-to-list 'company-backends 'company-ghc)
-      (add-hook 'haskell-mode-hook 'ghc-comp-init))))
+      (add-to-list 'company-backends 'company-ghc))))
```

Comment:
That's an unrelated issue, maybe we should have a separate PR for that?
The company-backends line was only changed because I had to reformat stuff :)